wolfcrypt/src/port/nrf51.c

```diff
@@ -171,13 +165,16 @@ static void rtc_config(void)
     uint32_t err_code;
 
     // Start the internal LFCLK XTAL oscillator
-    err_code = nrf_drv_clock_init(NULL);
+    err_code = nrf_drv_clock_init();
     APP_ERROR_CHECK(err_code);
 
-    nrf_drv_clock_lfclk_request();
+    nrf_drv_clock_lfclk_request(NULL);
 
     // Initialize RTC instance
-    err_code = nrf_drv_rtc_init(&rtc, NULL, rtc_handler);
+    nrf_drv_rtc_config_t config = NRF_DRV_RTC_DEFAULT_CONFIG;
```

Comment:
Variable declarations must be at top of function / brace. Some compilers don't like variable declaration in the middle.